mxcube3/routes/qutils.py

```diff
@@ -286,6 +286,11 @@ def _handle_dc(sample_id, node):
     parameters.pop('centred_position')
     queueID = node._node_id
     enabled, state = get_node_state(queueID)
+    try:
+        lims_data = mxcube.rest_lims.get_dc(node.id)
```

Comment:
There should probably not be a try except here since get_dc should return '' or similar
when we cant get data from lims. I guess you get an attribute error now because the
method does not exist on your lims object. ?